.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+jobs:
+  java:
+    machine:
+      image: circleci/classic:latest
+    working_directory: ~/misk
+    steps:
+      - checkout # check out source code to working directory
```

Comment:
nit: unnecessary comment

Reply:
eh this is helpful for me not knowing what checkout exactly does